ci/release/update-version.sh

```diff
@@ -56,3 +56,8 @@ for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
 done
 sed_runner "s/RAPIDS_VERSION_NUMBER=\".*/RAPIDS_VERSION_NUMBER=\"${NEXT_SHORT_TAG}\"/g" ci/build_docs.sh
+
+# Docs referencing source code
+for FILE in docs/source/*.rst; do
```

Comment:
This will only list the `.rst` files directly in that directory, so `docs/source/examples/ucx.rst` for example isn't included (it doesn't need to be though).

This could be updated to use `find` instead: https://github.com/rapidsai/cuml/blob/branch-24.06/ci/release/update-version.sh#L82

Reply:
I think?
```suggestion
find .docs/source/ -type f -name '*.rst' -print0 | while IFS= read -r -d '' FILE; do
```

Reply:
I don't think it's necessary right now but it won't hurt either, so I applied that change in https://github.com/rapidsai/dask-cuda/pull/1329/commits/19522fdc42e4685c9a30c87e6d52e7e63eaedcbe . Thanks for linking that `find` line.